data/overview.yaml

```diff
@@ -119,66 +119,66 @@ Groups:
 Rings:
   ring:
   Unitary ring:
-  ring morphisms:
-  subrings:
+  ring morphisms: 'algebra/ring.html#ring_hom'
+  subrings: 'ring_theory/subring.html#top'
   The ring Z on the integers:
   Product of rings:
-  Ideal of a commutative ring:
+  Ideal of a commutative ring: 'algebra/module.html#ideal'
   quotient rings:
   prime ideals:
-  maximal ideals:
+  maximal ideals: 'ring_theory/ideals.html#ideal.is_maximal'
   Chinese remainder theorem:
   Algebra:
     algebra over a commutative ring:
     associative algebra over a commutative ring:
   Divisibility in commutative integral rings:
-  Irreducible elements:
-  invertible elements:
+  Irreducible elements: 'algebra/associated.html#irreducible' 
+  Invertible elements: 'algebra/invertible.html#invertible'
   Coprime elements:
-  Unique factorization domain (UFD):
+  Unique factorisation domain (UFD): 'ring_theory/unique_factorization_domain.html'
   Greatest common divisor:
   Least common multiple:
   A[X] is a UFD when A is a UFD:
-  Principal ring:
+  Principal ring: 'ring_theory/principal_ideal_domain.html#submodule.is_principal'
   Bezouts theorem:
-  Euclidean rings:
+  Euclidean rings: 'algebra/euclidean_domain.html#euclidean_domain'
   Euclid's' algorithm:
   Z is an euclidean ring:
   K[X] is an euclidean ring when K is a field:
   Irreducible polynomial:
   Cyclotomic polynomials in Q[X]:
-  Eisenstein's criterion:
+  Eisenstein's criterion: 'number_theory/quadratic_reciprocity.html#zmod.eisenstein_lemma' # FIXME: cheat?
   Congruence in Z:
-  Prime numbers:
+  Prime numbers: 'algebra/associated.html#associates.prime' 
   Z/nZ and its invertible elements:
   Euler's indicator function (phi):
 Polynomial:
   Polynomial algebra on one or several indeterminates over a commutative ring:
-  Roots of a polynomial:
-  Multiplicity:
+  Roots of a polynomial: 'data/polynomial.html#polynomial.roots'
+  Multiplicity: 'data/polynomial.html#polynomial.root_multiplicity'
   Relationship between the coefficients and the roots of a split polynomial:
   Newton's sums:
   Derived polynomials:
   Decomposition into sums of homogeneous polynomials:
   Symmetric polynomials:
 Fields:
   fields:
-  subfields:
-  characteristic:
-  Frobenius morphism:
+  Characteristics: 'algebra/char_p.html#ring_char'
```

Comment:
Possible extra links: classes `char_p` and `char_zero`.

Reply:
I think this one makes a lot of sense

Reply:
Done